dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/NotificationService.kt

```diff
@@ -27,6 +28,7 @@ import java.util.UUID
  * Service that handles creation of notification events and sending notifications to interested parties
  * in case of elementary events
  */
+@Suppress("TooManyFunctions")
```

Comment:
Finde ich nicht gut. Kann man nicht ein paar thematisch sinnvoll zusammenhängende Funktionien in eine separate Klasse auslagern?